packaging/rpm/microshift.spec

```diff
@@ -358,6 +378,45 @@ cat assets/optional/multus/kustomization.x86_64.yaml >> %{buildroot}/%{_prefix}/
 mkdir -p -m755 %{buildroot}%{_datadir}/microshift/release
 install -p -m644 assets/optional/multus/release-multus-{x86_64,aarch64}.json %{buildroot}%{_datadir}/microshift/release/
 
+# kube-proxy
+install -d -m755 %{buildroot}/%{_prefix}/lib/microshift/manifests.d/000-microshift-kube-proxy
+# Copy all the manifests except the arch specific ones
+install -p -m644 assets/optional/kube-proxy/0* %{buildroot}/%{_prefix}/lib/microshift/manifests.d/000-microshift-kube-proxy
+install -p -m644 assets/optional/kube-proxy/kustomization.yaml %{buildroot}/%{_prefix}/lib/microshift/manifests.d/000-microshift-kube-proxy
+
+%ifarch %{arm} aarch64
+cat assets/optional/kube-proxy/kustomization.aarch64.yaml >> %{buildroot}/%{_prefix}/lib/microshift/manifests.d/000-microshift-kube-proxy/kustomization.yaml
+%endif
+
+%ifarch x86_64
+cat assets/optional/kube-proxy/kustomization.x86_64.yaml >> %{buildroot}/%{_prefix}/lib/microshift/manifests.d/000-microshift-kube-proxy/kustomization.yaml
+%endif
+
+# kube-proxy-release-info
+mkdir -p -m755 %{buildroot}%{_datadir}/microshift/release
+install -p -m644 assets/optional/kube-proxy/release-kube-proxy-{x86_64,aarch64}.json %{buildroot}%{_datadir}/microshift/release/
+
+# flannel
+install -d -m755 %{buildroot}/%{_prefix}/lib/microshift/manifests.d/000-microshift-flannel
+install -d -m755 %{buildroot}%{_sysconfdir}/systemd/system
+# Copy all the manifests except the arch specific ones
+install -p -m644 assets/optional/flannel/0* %{buildroot}/%{_prefix}/lib/microshift/manifests.d/000-microshift-flannel
+install -p -m644 assets/optional/flannel/kustomization.yaml %{buildroot}/%{_prefix}/lib/microshift/manifests.d/000-microshift-flannel
+install -p -m644 packaging/flannel/disableDefaultCNI.yaml %{buildroot}%{_sysconfdir}/microshift/config.d/disableDefaultCNI.yaml
```

Comment:
The target file should also be numbered in its prefix to allow for easier understanding of the order (e.g. `00-disable-default-cni.yaml`).
See https://github.com/openshift/microshift/blob/main/docs/user/howto_config.md#drop-in-configuration-directory

Reply:
Done